host/xtest/regression_4000.c

```diff
@@ -2724,6 +2727,7 @@ struct xtest_ac_case {
 	XTEST_AC_CASE(level, algo, mode, vect, XTEST_AC_ECDSA_UNION(vect))
 
 static const struct xtest_ac_case xtest_ac_cases[] = {
+#if defined(CFG_CRYPTOLIB_NAME_mbedtls) &&  defined(MBEDTLS_RSA_NO_CRT)
```

Comment:
Shouldn't this be `#if !defined(CFG_CRYPTOLIB_NAME_mbedtls) || defined(MBEDTLS_RSA_NO_CRT)`?

Reply:
Such modifications are more understandable,thanks.